english/got/got.md

```diff
@@ -0,0 +1,2 @@
+Who's excited about Game of Thrones Season 8?
+Then head [here](https://github.com/bhavesh-dhera/create-your-own-adventure/blob/master/english/marshmallow.md) and continue your work!
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.